### ISSUE TITLE
0.4508949222

### ISSUE THREAD

**Issue by oneonlee** [oneonlee/NLP-author-classification#4]
- batch_size = 16
- epochs = 6
- learning_rate = 5e-5
- **scheduler = torch.optim.lr_scheduler.StepLR(optimizer, step_size=1, gamma=0.1)**

epoch을 `3`(#3)에서 `6`으로 사용했을 때, 성능(LogLoss)이 `0.5092167652`(#3)에서 `0.4508949222`로 증가함
